packages/govern-console/src/containers/DAOSettings/DAOSettings.tsx

```diff
@@ -202,13 +215,11 @@ const DaoSettings: React.FC<DaoSettingFormProps> = () => {
       proof: proofCid,
     };
 
-    console.log('payload', payload, 'newConfig', newConfig);
-
     if (proposalInstance) {
       try {
         transactionsQueue.current = await proposalInstance.schedule(payload, buildConfig(config));
       } catch (error) {
-        console.log('Failed scheduling dao changes', error);
+        // console.log('Failed scheduling dao changes', error);
```

Comment:
Why leave it commented and not removing?

Reply:
should be removed.
i wish we had some sort of logger so it can only log when in staging or in local and not on production